.github/workflows/pipeline-segment-electron-test.yml

```diff
@@ -176,7 +176,7 @@ jobs:
           if [ "${{ inputs.is-asan }}" == "true" ]; then
             cd ..
             ASAN_SYMBOLIZE="$PWD/tools/valgrind/asan/asan_symbolize.py --executable-path=$PWD/out/Default/electron"
-            export ASAN_OPTIONS="symbolize=0 handle_abort=1"
+            export ASAN_OPTIONS="symbolize=0 handle_abort=1 detect_container_overflow=0"
```

Comment:
@VerteDinde this triages https://github.com/nodejs/node/issues/55584 - it's a false positive because not everything is instrumented with ASAN (in this case simdjson is not) and there will be a better fix upstream but in the meantime this prevent a crash for what's actually a valid read (see linked issue). Once a real fix is merged we can backport that and re-enable this